server/src/auth.ts

```diff
@@ -17,7 +18,48 @@ declare global {
   }
 }
 
-export default function setupAuth(app: express.Application) {
+export function setupMockAuth(app: express.Application) {
+
+  app.use(
+    session({
+      genid: (req) => uuid(),
+      secret: 'mock-secret',
+    })
+  );
+
+  app.use(passport.initialize());
+  app.use(passport.session());
+
+  // override values on this object to match what mock user you want
+  const customUserObject = {
+    id: '12345',
+    name: 'Adam Savage',
+    role: 'labbie'
+  }
+
+  passport.use(new MockStrategy({
+    // @ts-ignore
+    user: customUserObject
+  }));
+
+  app.get('/auth/mock',
+    passport.authenticate('mock'),
+    (req, res) => {
+      console.log('Mock Authenticating')
+      res.redirect('/')
+    });
+
+  passport.serializeUser(function (user, done) {
+    done(null, user);
+  });
+
+  passport.deserializeUser(function (user, done) {
+    // @ts-ignore
```

Comment:
lots of these cuz the library will only accept a specific user type that does not match our needs. Let me know if there is a better way